test/parallel/test-global.js

```diff
@@ -34,17 +34,21 @@ baseFoo = 'foo'; // eslint-disable-line no-undef
 global.baseBar = 'bar';
 
 assert.strictEqual(global.baseFoo, 'foo',
-                   'x -> global.x in base level not working');
+                   // eslint-disable-next-line no-undef
+                   `${baseFoo} -> global.baseFoo in base level not 
```

Comment:
This would result in very confusing message, IMO. I think you might be misunderstanding the original error message text. 

Try removing the eslint-disable-line comment that was added and using something like this for the message instead:

```js
`x -> global.x failed: global.baseDir = ${global.baseDir}`
```